libnopegl/src/backends/gl/pipeline_gl.c

```diff
@@ -375,22 +375,11 @@ int ngli_pipeline_gl_init(struct pipeline *s, const struct pipeline_params *para
 {
     struct pipeline_gl *s_priv = (struct pipeline_gl *)s;
 
-    s->type     = params->type;
```

Comment:
initilization / initialization typo in commit message

Reply:
Fixed.